app/src/main/java/com/gatheringhallstudios/mhworlddatabase/MainActivity.kt

```diff
@@ -49,7 +50,8 @@ class MainActivity : AppCompatActivity() {
             R.id.decorationListDestination,
             R.id.locationListDestination,
             R.id.charmListDestination,
-            R.id.weaponListDestination
+            R.id.weaponListDestination,
+            R.id.favoritesListDestination
```

Comment:
Did you mean to leave reference to favorites in the code? The PR says there are ~41 instances of `favorite` left but most can be knocked out with the refactor->rename tool since they're repeats.